### DISCUSSION TITLE
BOX64 Benchmark Results

### DISCUSSION THREAD

**Discussion by theofficialgman** [ptitSeb/box64#39] [General]
I'm creating a thread here to post box64 benchmarking results.
If you can do both a native benchmark and one through box64 that would be most interesting.

Geekbench 5 works through box64 and has a native arm64 build (you can find in on the geekbench discord)

Native performance vs Nintendo Switch Box64
Box64 commit: 928eb2a
https://browser.geekbench.com/v5/cpu/compare/6909003?baseline=8722188

LOTS of room for improvement especially in the AES-XTS benchmark

<img width="629" alt="Screenshot 2021-07-07 131525" src="https://user-images.githubusercontent.com/28281419/124801934-6b2fa280-df25-11eb-9e85-caabec599d84.png">

**Comment by theofficialgman** [comment]
updated results with box64 commit: d20d0079333a78b8013a9bcc09f1aaa294fec3ae
Nintendo Switch 2.091Ghz max clock
box64 d20d007 vs 928eb2a
https://browser.geekbench.com/v5/cpu/compare/8739220?baseline=8722188
![Screenshot from 2021-07-08 18-01-28](https://user-images.githubusercontent.com/28281419/124996231-98a84900-e016-11eb-9870-872957b01181.png)

native vs box64 d20d007
https://browser.geekbench.com/v5/cpu/compare/6909003?baseline=8739220
![Screenshot from 2021-07-08 18-01-40](https://user-images.githubusercontent.com/28281419/124996240-9ba33980-e016-11eb-9b99-efc83dfd3860.png)

**Comment by Heasterian** [comment]
@theofficialgman Could you also check perfomance difference of Switch in benchmark from this article? https://box86.org/2021/06/game-performances/
